src/Plugin/EntityBrowser/FieldWidgetDisplay/EntityLabel.php

```diff
@@ -36,4 +37,11 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
     return [];
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function isApplicable(EntityTypeInterface $entity_type) {
```

Comment:
Let's create a base plugin implementation and move this case (always TRUE) there.